api/clients/v2/accountant.go

```diff
@@ -5,115 +5,106 @@ import (
 	"fmt"
 	"math/big"
 	"sync"
+	"time"
 
 	disperser_rpc "github.com/Layr-Labs/eigenda/api/grpc/disperser/v2"
 	"github.com/Layr-Labs/eigenda/core"
 	"github.com/Layr-Labs/eigenda/core/meterer"
 	gethcommon "github.com/ethereum/go-ethereum/common"
 )
 
-var requiredQuorums = []core.QuorumID{0, 1}
-
 type Accountant struct {
 	// on-chain states
-	accountID         gethcommon.Address
-	reservation       *core.ReservedPayment
-	onDemand          *core.OnDemandPayment
-	reservationWindow uint64
-	pricePerSymbol    uint64
-	minNumSymbols     uint64
+	accountID    gethcommon.Address
+	reservations map[core.QuorumID]*core.ReservedPayment
+	onDemand     *core.OnDemandPayment
```

Comment:
why is onDemand not a per-quorum map? If its a current limitation then should be commented/documented here

Reply:
okay. onDemand is going to be only enabled on quorum 0. I was thinking we can update after we decide to support custom onDemand and design decentralized ratelimiting. updated comment in [commit](https://github.com/Layr-Labs/eigenda/pull/1664/commits/783ec1c7184a942bdfde1ef6ad4ffc611d96c776)

Reply:
Oh wait really? so onDemand not supported for EIGEN quorum?

Reply:
not quite, onDemand is supported in the way that EIGEN quorum can share the ETH quorum onDemand  balance. This logic is completely offchain. My comment above is the onchain configuration